content/talks/advanced-css-workshop.rst

```diff
@@ -0,0 +1,51 @@
+public: yes
+speakers: [miriam]
+workshop: true
+events:
+  - venue: 'Smashing Conference'
+    url: 'https://www.smashingconf.com/sf-2019/'
```

Comment:
It seems like this url isn't ever used? It seems odd to me that the link at the bottom of the page goes to the same page:

![screenshot 2018-12-10 12 49 26](https://user-images.githubusercontent.com/552316/49750918-1e569300-fc7a-11e8-8b30-ff9a9a1cd484.jpg)

Reply:
Related, this looks a bit odd (when you're giving two talks/workshops at the same conference) at the bottom of the talks page:

![screenshot 2018-12-10 12 49 14](https://user-images.githubusercontent.com/552316/49750948-33332680-fc7a-11e8-8fc0-83ef1797aa13.jpg)

Reply:
See if my latest addresses this well…